src/util/Histogram.h

```diff
@@ -39,6 +39,9 @@
 // Third party headers
 #include <nlohmann/json.hpp>
 
+// qserv headers
+#include "global/constants.h"
```

Comment:
Please, see my comments posted earlier for this header. Definitions that you are importing here should be placed into the dedicated header:
```
#include "global/clock_defs.h"
```
or something.